x-pack/plugins/observability_solution/observability_onboarding/public/application/onboarding_flow_form/onboarding_flow_form.tsx

```diff
@@ -0,0 +1,277 @@
+              }
+            )}
+          </li>
+          <li>
+            {i18n.translate(
+              'xpack.observability_onboarding.onboardingFlowForm.li.identifyPerformanceBottlenecksWithLabel',
+              {
+                defaultMessage:
+                  'Identify performance bottlenecks with automated and curated visual representation of all dependencies.',
+              }
+            )}
+          </li>
+          <li>
+            {i18n.translate(
+              'xpack.observability_onboarding.onboardingFlowForm.li.drillIntoAnomaliesTransactionLabel',
+              {
+                defaultMessage:
+                  'Drill into anomalies, transaction details, errors and metrics for deeper analysis.',
+              }
+            )}
+          </li>
+          <li>
+            {i18n.translate(
+              'xpack.observability_onboarding.onboardingFlowForm.li.useMachineLearningToLabel',
+              {
+                defaultMessage:
+                  'Use machine learning to automatically detect anomalies.',
+              }
+            )}
+          </li>
+        </ul>
+      ),
+    },
+    {
+      id: 'infra',
+      label: i18n.translate(
+        'xpack.observability_onboarding.experimentalOnboardingFlow.euiCheckableCard.monitorMyInfrastructureLabel',
+        { defaultMessage: 'Monitor my infrastructure' }
+      ),
+      description: (
+        <ul>
+          <li>
+            {i18n.translate(
+              'xpack.observability_onboarding.onboardingFlowForm.li.builtOnPowerfulElasticsearchLabel',
+              {
+                defaultMessage:
+                  'Built on powerful Elasticsearch, stream in and scale infrastructure metrics from your systems, cloud, network, and other infrastructure sources',
+              }
+            )}
+          </li>
+          <li>
+            {i18n.translate(
+              'xpack.observability_onboarding.onboardingFlowForm.li.getLogicalAndPhysicalLabel',
+              {
+                defaultMessage:
+                  'Get logical and physical views of your infrastructure topology.',
+              }
+            )}
+          </li>
+          <li>
+            {i18n.translate(
+              'xpack.observability_onboarding.onboardingFlowForm.li.breakDownApplicationAndLabel',
+              {
+                defaultMessage:
+                  'Break down application and infrastructure silos for faster root cause detection',
+              }
+            )}
+          </li>
+        </ul>
+      ),
+    },
+  ];
+
+  const [selectedId, setSelectedId] = useState<string>();
+  const [hoveredId, setHoveredId] = useState<string>();
+
+  const visibleOption = hoveredId
+    ? options.find((option) => option.id === hoveredId)
+    : selectedId
+    ? options.find((option) => option.id === selectedId)
+    : undefined;
+
+  return (
+    <EuiPanel hasBorder>
+      <TitleWithIcon
+        iconType="indexRollupApp"
```

Comment:
is this the right icon? Seems weird to reuse something that has a very specific meaning already.

Reply:
The icon is as per the latest design. I don't think we have icons specific to this use case so they're al going to be a bit made up.

Reply:
makes sense, I guess it's fine to start this way.